unit-tests/py/rspy/file.py

```diff
@@ -0,0 +1,124 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+import os, re, platform, subprocess, sys
+
+# Remove Python's default list of places to look for modules so that we use our log module
+sys.path = list()
+sys.path.append( '' )  # directs Python to search modules in the current directory first
+sys.path.append( os.path.dirname( sys.executable ))
+sys.path.append( os.path.join( os.path.dirname( sys.executable ), 'DLLs' ))
+sys.path.append( os.path.join( os.path.dirname( sys.executable ), 'lib' ))
+# Add our py/ module directory
+current_dir = os.path.dirname( os.path.abspath( __file__ ))
+sys.path.append( os.path.dirname( current_dir ) )
+
+from rspy import log
+
+# get os and directories for future use
+# NOTE: WSL will read as 'Linux' but the build is Windows-based!
+system = platform.system()
+if system == 'Linux'  and  "microsoft" not in platform.uname()[3].lower():
+    linux = True
+else:
+    linux = False
+
+def filesin( root ):
```

Comment:
I think we need to rename this... `for f in file.filesin(...)` isn't so good... how about `file.inside_dir(...)`?

Reply:
as you wish